c2corg_ui/templates/article/helpers/view.html

```diff
@@ -31,10 +31,48 @@
       <p><span class="value-title" translate>article_type</span>: <span class="value" x-translate>${article['article_type']}</span></p>
     % endif
 
+    % if article.get('author') and article['article_type'] != 'collab' and article['author']['user_id'] != article['associations']['users'][0]['document_id']:
```

Comment:
> But if there is only one associated user but he isn't the creator, he must be shown as the author
I think this is the reason why I added it. It is confusing when to show what. Should I change/delete it to keep more simple?